### DISCUSSION TITLE
new wizard

### DISCUSSION THREAD

**Discussion by ronbon172** [EdgeTX/edgetx#5625] [Ideas]
Hello,
would it be possible to create a new wizard or update the existing ones with a new feature. I have several Spektrum models with dsmx receivers and safe technology, and can't get the programming right. having a wizard with the Spektrum receiver options to mimic Spektrum transmitters would be great!

**Comment by pfeerick** [comment]
Possible? yes. Likely? not unless someone wants to do it 😆 The wizards have been updated a little bit recently, but probably still not anything like what you are asking about. Plus, you would probably need to be a lot clearer as to exactly what changes you would like to see before anyone was to take it on...

For Spektrum, depending on the receiver, a lot of the programming is done via forward programming now, so perhaps https://github.com/frankiearzu/DSMTools is of interest.
